x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/license_callout/license_callout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import { EuiPanel, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+
+import { LicensingLogic } from '../../../shared/licensing';
+import { EuiButtonTo } from '../../../shared/react_router_helpers';
+
+import { PRODUCT_SELECTOR_CALLOUT_HEADING } from '../../constants';
+
+import { LICENSE_CALLOUT_BODY, LICENSE_CALLOUT_BUTTON } from './constants';
+
+export const LicenseCallout: React.FC = () => {
+  const { hasPlatinumLicense, isTrial } = useValues(LicensingLogic);
+
+  if (hasPlatinumLicense && !isTrial) return null;
+
+  return (
+    <EuiPanel hasShadow={false} hasBorder className="productCard" paddingSize="l">
+      <EuiFlexGroup gutterSize="s" alignItems="center" justifyContent="spaceBetween">
+        <EuiFlexItem grow={7}>
+          <EuiText>
+            <h3>{PRODUCT_SELECTOR_CALLOUT_HEADING}</h3>
+          </EuiText>
+          <EuiText size="s">{LICENSE_CALLOUT_BODY}</EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={1}>
+          <div />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <div>
+            <EuiButtonTo to="/app/management/stack/license_management" shouldNotCreateHref>
+              {LICENSE_CALLOUT_BUTTON}
+            </EuiButtonTo>
+          </div>
+        </EuiFlexItem>
```

Comment:
Cleanup suggestion for pruning unnecessary `<div>`s (does not affect spacing AFAICT)

```suggestion
        <EuiFlexItem grow={1} />
        <EuiFlexItem grow={false}>
          <EuiButtonTo to="/app/management/stack/license_management" shouldNotCreateHref>
            {LICENSE_CALLOUT_BUTTON}
          </EuiButtonTo>
        </EuiFlexItem>
```

Reply:
Thanks again!